src/Controller/GameController.php

```diff
@@ -10,51 +10,75 @@
 
 use App\Model\Troop;
 use App\Model\TroopManager;
+use App\Model\Castle;
+use App\Model\CastleManager;
 
 class GameController extends AbstractController
 {
     private $troopManager;
+    private $castleManager;
 
     public function __construct()
     {
         parent::__construct();
         $this->troopManager = new TroopManager();
+        $this->castleManager = new CastleManager();
     }
 
     public function init(): string
     {
+        // Check if access to the database and data deletion
         if (false === $this->troopManager->deleteAll()) {
-            header("HTTP/1.0 404 Not Found");
-            echo '404 - Page not found';
+            header("HTTP/1.1 503 Service Unavailable");
+            echo '503 - Service Unavailable';
```

Comment:
You should use a return end the controller after a problem
You shouldn't echo in the controller, return you page instead.

Reply:
I created a redirect to a 503 page